### ISSUE TITLE
Stream and record local video

### ISSUE THREAD

**Issue by bocklucas** [HackerHouseYT/Smart-Security-Camera#7]
Would it be possible to create a method that also creates a local video the same as a live stream so in the even the network goes down you can still have a copy on the Pi?

Thanks!

**Comment by debsahu** [comment]
Yes easily, read https://picamera.readthedocs.io/en/release-1.13/recipes1.html#recording-video-to-a-file But I'm not sure how SD card will effect your Pi Zero's performance.

**Comment by curlydlamini** [comment]
Would it be possible to automatically start main.py

**Comment by debsahu** [comment]
Check my fork, https://github.com/debsahu/Smart-Security-Camera I have added system files and instructions.
